tests/tests_app/core/test_lightning_work.py

```diff
@@ -146,7 +146,7 @@ def run(self):
     class BlockingQueue(MockQueue):
         """A Mock for the file copier queues that keeps blocking until we want to end the thread."""
 
-        keep_blocking = True
+        keep_blocking = False
```

Comment:
if you change that, then you may as well just go with the regular MockQueue :)

Reply:
Yes, it is fine if the file copier terminates.